Trial division Algorithm for Prime Factorization.txt

```diff
@@ -1,50 +1,14 @@
-// CPP implementation of 
+
 // Trial Division Algorithm 
 #include <bits/stdc++.h> 
```

Comment:
remove this as well.

Reply:
Remove the include also, as the code which will use this snippet will already have one "include" line.